### PR TITLE
Added some a11y recommendation as per #13436

### PR DESCRIPTION
**Pull request by rtripault**
### What does it do?

Added some a11y recommendation as per #13436

### CODE REVIEW

File: manager/templates/default/header.tpl [hunk 1]
```diff
@@ -76,12 +76,12 @@
...
             </ul>
             {if $_search}
-            <div id="modx-manager-search"></div>
+            <div id="modx-manager-search" role="search"></div>
             {/if}
         </div>
     </div>
         <div id="modAB"></div>
         <div id="modx-leftbar"></div>
-		<div id="modx-action-buttons-container"></div>
```
Comment (rtripault) [inline manager/templates/default/header.tpl]: Those were tabs... TABS!

File: manager/assets/modext/widgets/core/modx.searchbar.js [hunk 2]
```diff
@@ -11,12 +11,19 @@ MODx.SearchBar = function(config) {
...
-        // ,triggerConfig: { // handled globally for Ext.form.ComboBox via override
-        //     tag: 'span'
-        //     ,cls: 'x-form-trigger icon icon-large icon-search'
-        // }
-        // ,shadow: false // handled globally for Ext.form.ComboBox via override
-        // ,triggerAction: 'query'
+        ,triggerConfig: {
+            tag: 'button'
+            ,type: "submit"
+            ,"aria-label": "Go"
```
Comment (rtripault) [inline manager/assets/modext/widgets/core/modx.searchbar.js]: Probably needs some lexicon string here instead

**Comment by Mark-H** [comment]
@jpdevries Could you check this pull request to verify it achieves what you suggested in #13436?

**Comment by jpdevries** [comment]
Looks good @rtripault. I think the only thing we could do more is wrap the input and the button in an actual `<form>`. I can't remember, maybe that wasn't feasible. It looks like I didn't mention it in https://github.com/modxcms/revolution/issues/13436

**Pull request merged**